### ISSUE TITLE
[Preview-MAS Violation]:Screen reader is narrating caption informaton twice when we press shortcut key "T"

### ISSUE THREAD

**Issue by NavyaGoberu** [nvaccess/nvda#9620]
User Experience:
User who rely on screen reader if screen reader is narrating repeated information user will confuse.

Test Environment:
OS: Windows 10
Browser: Chrome
Screen reader: NVDA
URL: https://microsoftignite-staging-beta.azurewebsites.net/en-us/ignite/pricing

Repro Steps:
Step 1: Open the above URL using any browser.
Step 2: navigate to hero section and Press “T” to read the caption information of the table.
Step 3: Verify screen reader is narrating caption information or not?

Actual:
Screen reader caption information twice when we press “T” to read the table info.
Expected:
Screen reader should be narrate the captain single time.

MAS Reference:
MAS 4.2.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

**Comment by DrSooom** [comment]
@NavyaGoberu: Please specify the Win10, the Google Chrome and the NVDA version in the issue description. Thanks.

**Comment by Adriani90** [comment]
Duplicate of #9914
